docs/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,11 @@
+### `cosmos/cosmos-sdk` Release Process
+
+- [ ] 1. Decide on release designation (are we doing a patch, or minor version bump) and start a P.R. for the release
+- [ ] 2. Add commits/PRs that are desired for this release **that haven’t already been added to develop**
+- [ ] 3. Ensure that `CHANGELOG.md` contains links to issues/PRs for each item
+- [ ] 4. Summarize breaking API changes section under “Breaking Changes” section to the `CHANGELOG.md` to bring attention to any breaking API changes that affect RPC consumers
```

Comment:
Can we come up with a format for this in coordination with Voyager to make sure we're communicating clearly?

Reply:
Brainstorming:
```
### Breaking Changes
#### API
- Validator struct changed. Renamed `slashes` to `punishments`
  - Affected endpoints:
    - /validatorsets
    - /stake/candidates
#### CLI
- Renamed delegation cmd `gaiacli stake delegation -> gaiacli stake bond`
```

Reply:
Like this!!